ServerCore/Properties/launchSettings.json

```diff
@@ -12,16 +12,20 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "ASPNETCORE_HOSTINGSTARTUP__KEYVAULT__CONFIGURATIONENABLED": "true",
+        "ASPNETCORE_HOSTINGSTARTUP__KEYVAULT__CONFIGURATIONVAULT": "https://PuzzleServerTestKeyVault.vault.azure.net"
       }
     },
     "ServerCore": {
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
```

Comment:
Do you need to leave this on for now? If so, please file an issue to turn it off

Reply:
This is the only way to get useful error messages right now. I'm looking into other ways for the long term (supposedly I can turn on logging in the web.config, but the web.config is generated during Publish & I haven't traced that back to what I actually need to do). I'll add an issue.